src/RubiksCube.ts

```diff
@@ -1,27 +1,75 @@
+import Color from './color';
 import {RubiksCubeStateStructure} from './RubiksCubeStateStructure'
-import {BlockState} from './blockStateStructure'
+import TurnService from './TurnService'
 
 class RubiksCube {
   public readonly state: RubiksCubeStateStructure
-  private blockState: BlockState
+  public readonly turnService: TurnService
+  public upSideVals = [0,1,2]
+  public leftSideVals = [0,3,6]
+  public downSideVals = [6,7,8]
+  public rightSideVals = [2,5,8]
```

Comment:
The naming isn't clear. Should be smth like:
* sideUpRowIndexes
* sideLeftRowIndexes
* ....